### PR TITLE
[Sprint 11] Fix notification UI

### PR DESCRIPTION
**Pull request by nguyenminhduong**
Fix notification UI

### CODE REVIEW

File: app/src/main/java/com/framgia/wsm/utils/common/TimeAgo.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import com.framgia.wsm.R;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by minhd on 7/31/2017.
+ */
+
+public class TimeAgo {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/utils/common/TimeAgo.java]: WARNING: Utility classes should not have a public or default constructor.

File: app/src/main/java/com/framgia/wsm/utils/common/TimeAgo.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    public static String toDuration(long duration, Context context) {
+        String[] timesString = context.getResources().getStringArray(R.array.time_string);
+        StringBuilder res = new StringBuilder();
+        for (int i = 0; i < TimeAgo.TIMES.size(); i++) {
+            Long current = TimeAgo.TIMES.get(i);
+            long temp = duration / current;
+            if (temp > 0) {
+                res.append(temp)
+                        .append(" ")
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/utils/common/TimeAgo.java]: WARNING: [AppendCharacterWithChar] Avoid appending characters as strings in StringBuffer.append.
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AppendCharacterWithChar

File: app/src/main/java/com/framgia/wsm/utils/common/TimeAgo.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                        .append(timesString[i])
+                        .append(temp != 1 ? context.getString(R.string.s) : "")
+                        .append(" ")
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/utils/common/TimeAgo.java]: WARNING: [AppendCharacterWithChar] Avoid appending characters as strings in StringBuffer.append.
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/strings.html#AppendCharacterWithChar

File: app/src/main/res/values/array.xml [hunk 4]
```diff
@@ -17,4 +17,13 @@
         <item>am</item>
         <item>pm</item>
     </string-array>
+
+    <string-array name="time_string">
```
Comment (trainer-android-fr) [inline app/src/main/res/values/array.xml]: FATAL: [MissingTranslation] "`time_string`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -280,4 +280,8 @@
     <string name="delete_form_success">Delete form success!</string>
     <string name="this_account_has_been_login_in_another_device">This account has been login in another device.</string>
     <string name="view_detail">View Detail</string>
+
+    <string name="ago">ago</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`ago`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -280,4 +280,8 @@
...
+    <string name="just_now">just now</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`just_now`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -280,4 +280,8 @@
...
+    <string name="s">s</string>
```
Comment (trainer-android-fr) [inline app/src/main/res/values/strings.xml]: FATAL: [MissingTranslation] "`s`" is not translated in "ja" (Japanese)
 If an application has more than one locale, then all the strings declared in one language should also be translated in all other languages.

If the string should *not* be translated, you can add the attribute `translatable="false"` on the `<string>` element, or you can define all your non-translatable strings in a resource file called `donottranslate.xml`. Or, you can ignore the issue with a `tools:ignore="MissingTranslation"` attribute.

By default this detector allows regions of a language to just provide a subset of the strings and fall back to the standard language strings. You can require all regions to provide a full translation by setting the environment variable `ANDROID_LINT_COMPLETE_REGIONS`.

You can tell lint (and other tools) which language is the default language in your `res/values/` folder by specifying `tools:locale="languageCode"` for the root `<resources>` element in your resource file. (The `tools` prefix refers to the namespace declaration `http://schemas.android.com/tools`.)

File: app/src/main/java/com/framgia/wsm/utils/common/TimeAgo.java [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+import com.framgia.wsm.R;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by minhd on 7/31/2017.
+ */
+
+public final class TimeAgo {
```
Comment (trainer-android-fr) [inline app/src/main/java/com/framgia/wsm/utils/common/TimeAgo.java]: WARNING: Utility classes should not have a public or default constructor.

File: app/src/main/java/com/framgia/wsm/screen/notification/NotificationAdapter.java [hunk 9]
```diff
@@ -16,10 +16,12 @@
 public class NotificationAdapter
         extends BaseRecyclerViewAdapter<NotificationAdapter.ItemViewHolder> {
     private final List<Notification> mNotifications;
+    private Context mContext;
     private OnRecyclerViewItemClickListener<Notification> mItemClickListener;
 
     public NotificationAdapter(@NonNull Context context) {
         super(context);
+        mContext = context;
```
Comment (daolq3012) [inline app/src/main/java/com/framgia/wsm/screen/notification/NotificationAdapter.java]: pls change to 
```java
mContext = context.getAplicationContext();
```

**Pull request merged**